src/Components/Publishing/Layouts/__tests__/NewsLayout.test.tsx

```diff
@@ -24,4 +24,12 @@ describe("News Layout", () => {
     component.simulate("click")
     expect(component.find(NewsSectionContainer).length).toEqual(9)
   })
+
+  it("looks right on mobile", () => {
```

Comment:
Maybe -> `renders the news layout on mobile` ? Feels a little more consistent and readable.